conda/cli/install.py

```diff
@@ -493,24 +492,32 @@ def install_revision(args, parser):
     for repodata_fn in Repodatas(repodata_fns, index_args):
         with repodata_fn as repodata:
             with get_spinner(f"Collecting package metadata ({repodata})"):
-                index = get_index(
-                    channel_urls=index_args["channel_urls"],
-                    prepend=index_args["prepend"],  # --override-channels
+                index = Index(
+                    channels=index_args["channel_urls"],
+                    prepend=index_args["prepend"],
                     platform=None,
-                    use_local=index_args["use_local"],  # --use-local
-                    # use_cache=index_args["use_cache"],  # --use-index-cache
-                    # unknown=index_args["unknown"],  # --unknown
+                    use_local=index_args["use_local"],
```

Comment:
```suggestion
                    # these options were commented out in the version of this
                    # code bit that used the now-deprecated `get_index` function
                    # we have left them here so that this information is not lost
                    # use_cache=index_args["use_cache"],  # --use-index-cache
                    # unknown=index_args["unknown"],  # --unknown
                    use_local=index_args["use_local"],
```